.github/workflows/sbt-node-snyk.yml

```diff
@@ -0,0 +1,40 @@
+name: SBT Node Snyk
+
+on:
+  workflow_call:
+    inputs:
+      DEBUG:
+        type: string
+        required: false
+    secrets:
+      SNYK_TOKEN:
+        required: true
+
+jobs:
+    security:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout branch
+              uses: actions/checkout@v2
+
+            - name: Setup debug var
+              run: echo INPUT_DEBUG=${{ inputs.DEBUG }} >> $GITHUB_ENV
+              shell: bash
+
+            - name: Get node version
+              run: echo NODE_VERSION=$(cat .nvmrc) >> $GITHUB_ENV
+
+            - uses: snyk/actions/setup@0.3.0
+            - uses: actions/setup-node@v2
+              with:
+                  node-version: ${{ env.NODE_VERSION }}
```

Comment:
Since https://github.com/actions/setup-node/pull/338 the `setup-node` action supports reading of `.nvmrc` files (among others). This can be simplified to:

```yaml
- uses: actions/setup-node@v2
  with:
    node-version-file: '.nvmrc'
```

It's worth noting that we're requiring `.nvmrc` to exist at the root of the repository. This is typically true, but not always.

Reply:
That's great! I had missed that, thanks @akash1810!

Reply:
That does causes projects with a missing `.nvmrc` to fail. I could either change it back to how it was previously, which will mean it defaults to the [PATH version](organisation), or add another to use the LTS if the version file is missing.

Reply:
I'd keep it simple and make it a requirement to have the file. We always want to specify the node version to ensure a deterministic build, and we want to do that via a file in VCS (as opposed to a TeamCity setting).

Reply:
Excellent point, I'll leave this workflow as is and update... "workflow" to include an `.nvmrc` file.